### PR TITLE
Extended default callouts for new journeys

### PR DESCRIPTION
**Pull request by techsmyth**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

**Comment by larssondenise** [comment]
@techsmyth to check! can't make you reviewer as you created the PR

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
These changes are indeed very nice but will be felt by like 1% of the cases unless we have a data migration, check whether this data exists, and if not - add it.

**Comment by techsmyth** [comment]
@valentinyanakiev afraid not our rights to add them into existing spaces etc. but we can add callout templates to the library to make them more accessible & maybe do some handholding with key spaces…

**Comment by techsmyth** [comment]
@valentinyanakiev please bring this one down to develop if it is working ok. The story about getting existing spaces / challenges updated should be raised as a separate story - and it is a non-trivial one as we cannot just go and modify existing spaces / challenges with new data.

**Review by valentinyanakiev — DISMISSED** [review]
These templates fail the most basic test - creating a Space:

```
{
  "errors": [
    {
      "message": "Please provide a whiteboard",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "createSpace"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "stacktrace": [
            "Error: Please provide a whiteboard",
            "    at CalloutService.createCallout (/home/valentin/work/repos/alkemio/server/src/domain/collaboration/callout/callout.service.ts:90:13)",
            "    at CollaborationService.addDefaultCallouts (/home/valentin/work/repos/alkemio/server/src/domain/collaboration/collaboration/collaboration.service.ts:102:49)",
            "    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
            "    at SpaceService.createSpace (/home/valentin/work/repos/alkemio/server/src/domain/challenge/space/space.service.ts:141:27)",
            "    at SpaceResolverMutations.createSpace (/home/valentin/work/repos/alkemio/server/src/domain/challenge/space/space.resolver.mutations.ts:70:19)"
          ]
        }
      }
    }
  ],
  "data": null
}
```

@larssondenise I'd expect this to be tested locally before this is moved to review:
- creating a new space
- creating a new challenge inside a new / old space
- creating a new opportunity inside a new / old challenge

**Comment by techsmyth** [comment]
@valentinyanakiev they should now be creating ok.

**Comment by valentinyanakiev** [comment]
> @valentinyanakiev they should now be creating ok.

@techsmyth A super quick test and it fails on the first attempt:

```
{
  "errors": [
    {
      "message": "CreatedBy not set on Whiteboard with id 057d3b0f-12a3-4c8c-a19c-1f64d9de2a0a",
      "locations": [
        {
          "line": 194,
          "column": 3
        }
      ],
      "path": [
        "space",
        "challenge",
        "collaboration",
        "callouts",
        1,
        "whiteboards",
        0,
        "createdBy"
      ],
      "extensions": {
        "code": "ENTITY_NOT_INITIALIZED",
        "exception": {
          "message": "CreatedBy not set on Whiteboard with id 057d3b0f-12a3-4c8c-a19c-1f64d9de2a0a",
          "error": "CreatedBy not set on Whiteboard with id 057d3b0f-12a3-4c8c-a19c-1f64d9de2a0a",
          "context": "collaboration",
          "code": "ENTITY_NOT_INITIALIZED",
          "stacktrace": [
            "GraphQLError: CreatedBy not set on Whiteboard with id 057d3b0f-12a3-4c8c-a19c-1f64d9de2a0a",
            "    at new BaseException (/home/valentin/work/repos/alkemio/server/src/common/exceptions/base.exception.ts:10:5)",
            "    at new EntityNotInitializedException (/home/valentin/work/repos/alkemio/server/src/common/exceptions/entity.not.initialized.exception.ts:6:5)",
            "    at WhiteboardResolverFields.createdBy (/home/valentin/work/repos/alkemio/server/src/domain/common/whiteboard/whiteboard.resolver.fields.ts:42:13)",
            "    at /home/valentin/work/repos/alkemio/server/node_modules/@nestjs/core/helpers/external-context-creator.js:70:33",
            "    at runMicrotasks (<anonymous>)",
            "    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
            "    at Object.target [as createdBy] (/home/valentin/work/repos/alkemio/server/node_modules/@nestjs/core/helpers/external-context-creator.js:77:28)"
          ]
        }
      }
    }
  ],
```

Maybe I should have mentioned - please, after creating the Journeys, try to open them. It fails on every level.

**Comment by techsmyth** [comment]
@valentinyanakiev ok I thought I had done that but clearly missed something. @alstojanovic as Valentin is away tomorrow can you please take a look at the latest fix?

**Comment by alstojanovic** [comment]
Sure, will take a look.

File: src/domain/challenge/challenge/challenge.default.callouts.ts [hunk 1]
```diff
@@ -39,44 +42,41 @@ export const challengeDefaultCallouts: CreateCalloutInput[] = [
...
+      displayName: 'Relevant news, research or use cases 📰',
       description:
-        '✍️ Contribute your insights to understanding the context. It is about surfacing up the wisdom of the community. Add your own post, or comment on posts added by others.',
+        'Please share any relevant insights to help us better understand the Challenge. You can describe why it is relevant and add a link or upload a document with the article. You can also comment on the insights already submitted by other community members!',
       tagsets: [
         {
           name: TagsetReservedName.CALLOUT_DISPLAY_LOCATION,
           type: TagsetType.SELECT_ONE,
-          tags: [ChallengeDisplayLocation.CONTRIBUTE_RIGHT],
+          tags: [ChallengeDisplayLocation.CONTRIBUTE],
```
Comment (alstojanovic) [inline src/domain/challenge/challenge/challenge.default.callouts.ts:76]: This one is located on the left but might be a client issue.

**Review by alstojanovic — APPROVED** [review]
Tested crating space, challenge and opportunity. Default callouts are in place in regards to new changes.

**Pull request merged**